### DISCUSSION TITLE
Leitura / emissão CTeOs modelo 67

### DISCUSSION THREAD

**Discussion by Markosbn** [wmixvideo/nfe#860] [Geral]
Bom dia prezados, gostaria de saber se a biblioteca esta preparada para processar xml da CTeOS, que seria  o CTe modelo 67. Dei uma olhada na estrutura hoje do processamento do CTe, e não localizei alguns campos extras que possuem no modelo 67, por conta disto fiquei em duvida se a lib consegue fazer este procedimento também.

**Comment by fincatto** [comment]
Bom dia @Markosbn. 
Eu particularmente não uso CTe. Mas você pode implementar os campos faltantes e mandar um PR que a gente adicionar na lib. 
PS: é importante mandar as informações da nota tecnica onde constam as alterações e os testes unitários, para que o revisor consiga testar e avaliar apropriadamente.
